hadoop-ozone/dev-support/checks/acceptance.sh

```diff
@@ -41,6 +41,7 @@ export OZONE_ACCEPTANCE_SUITE
 cd "$DIST_DIR/compose" || exit 1
 ./test-all.sh 2>&1 | tee "${REPORT_DIR}/output.log"
 RES=$?
-cp result/* "$REPORT_DIR/"
+cp -rv result/* "$REPORT_DIR/"
 cp "$REPORT_DIR/log.html" "$REPORT_DIR/summary.html"
+find "$REPORT_DIR" -type f -empty -print0 | xargs -0 rm -v
```

Comment:
Did we observe a lot of empty files in the output? (Examples would help me understand)

IMO it's fine to keep those empty files that could make debugging the scripts easier (e.g. HDDS-6327, to know if the output is empty or just not copied over).
The empty file **names** could also hint devs new to the GH action result what they should expect from the log bundles.
If there are specific empty files we want to remove we'd better remove them explicitly, rather than removing all of them? Just my two cents.

Either way works for me.

Reply:
To clarify: this is mostly for datanode audit log files (and OM audit log in certain tests).

Reply:
You are right. `-v` does the trick ;)